src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Legacy/CSharpTokenizer.cs

```diff
@@ -10,95 +10,96 @@
+            { "null", CSharpSyntaxKind.NullKeyword },
+            { "sizeof", CSharpSyntaxKind.SizeOfKeyword },
+            { "typeof", CSharpSyntaxKind.TypeOfKeyword },
+            { "implicit", CSharpSyntaxKind.ImplicitKeyword },
+            { "private", CSharpSyntaxKind.PrivateKeyword },
+            { "this", CSharpSyntaxKind.ThisKeyword },
+            { "using", CSharpSyntaxKind.UsingKeyword },
+            { "extern", CSharpSyntaxKind.ExternKeyword },
+            { "return", CSharpSyntaxKind.ReturnKeyword },
+            { "stackalloc", CSharpSyntaxKind.StackAllocKeyword },
+            { "uint", CSharpSyntaxKind.UIntKeyword },
+            { "base", CSharpSyntaxKind.BaseKeyword },
+            { "catch", CSharpSyntaxKind.CatchKeyword },
+            { "continue", CSharpSyntaxKind.ContinueKeyword },
+            { "double", CSharpSyntaxKind.DoubleKeyword },
+            { "for", CSharpSyntaxKind.ForKeyword },
+            { "in", CSharpSyntaxKind.InKeyword },
+            { "lock", CSharpSyntaxKind.LockKeyword },
+            { "object", CSharpSyntaxKind.ObjectKeyword },
+            { "protected", CSharpSyntaxKind.ProtectedKeyword },
+            { "static", CSharpSyntaxKind.StaticKeyword },
+            { "false", CSharpSyntaxKind.FalseKeyword },
+            { "public", CSharpSyntaxKind.PublicKeyword },
+            { "sbyte", CSharpSyntaxKind.SByteKeyword },
+            { "throw", CSharpSyntaxKind.ThrowKeyword },
+            { "virtual", CSharpSyntaxKind.VirtualKeyword },
+            { "decimal", CSharpSyntaxKind.DecimalKeyword },
+            { "else", CSharpSyntaxKind.ElseKeyword },
+            { "operator", CSharpSyntaxKind.OperatorKeyword },
+            { "string", CSharpSyntaxKind.StringKeyword },
+            { "ulong", CSharpSyntaxKind.ULongKeyword },
+            { "bool", CSharpSyntaxKind.BoolKeyword },
+            { "char", CSharpSyntaxKind.CharKeyword },
+            { "default", CSharpSyntaxKind.DefaultKeyword },
+            { "foreach", CSharpSyntaxKind.ForEachKeyword },
+            { "long", CSharpSyntaxKind.LongKeyword },
+            { "void", CSharpSyntaxKind.VoidKeyword },
+            { "enum", CSharpSyntaxKind.EnumKeyword },
+            { "finally", CSharpSyntaxKind.FinallyKeyword },
+            { "int", CSharpSyntaxKind.IntKeyword },
+            { "out", CSharpSyntaxKind.OutKeyword },
+            { "sealed", CSharpSyntaxKind.SealedKeyword },
+            { "true", CSharpSyntaxKind.TrueKeyword },
+            { "goto", CSharpSyntaxKind.GotoKeyword },
+            { "unchecked", CSharpSyntaxKind.UncheckedKeyword },
+            { "interface", CSharpSyntaxKind.InterfaceKeyword },
+            { "break", CSharpSyntaxKind.BreakKeyword },
+            { "checked", CSharpSyntaxKind.CheckedKeyword },
+            { "namespace", CSharpSyntaxKind.NamespaceKeyword },
+            { "when", CSharpSyntaxKind.WhenKeyword },
+            { "where", CSharpSyntaxKind.WhereKeyword }
         };
```

Comment:
Consider adding a test or assert that checks those match (stringifying the enum value gives back a string equal to the key)